libs/sdk-ui-dashboard/src/presentation/filterBar/attributeFilter/DefaultDashboardAttributeFilter.tsx

```diff
@@ -26,40 +45,116 @@ export const DefaultDashboardAttributeFilter = (props: IDashboardAttributeFilter
     const { parentFilters, parentFilterOverAttribute } = useParentFilters(filter);
     const locale = useDashboardSelector(selectLocale);
     const attributeFilter = useMemo(() => dashboardAttributeFilterToAttributeFilter(filter), [filter]);
-    const theme = useTheme();
+    // const theme = useTheme();
+    const [isConfigurationOpen, setIsConfigurationOpen] = useState(false);
+
+    const CustomDropdownButton = useMemo(() => {
+        return function DropdownButton(props: IAttributeFilterDropdownButtonProps) {
```

Comment:
here memo is not necessary it could be simple component 

const  CustomDropdownButton: React.FC<IAttributeFilterDropdownButtonProps> = (props)=> <AttributeFilterDropdownButton {...props} isDraggable={isDraggable} />;